unitOneAssessmentPractice.js

```diff
@@ -4,39 +4,96 @@ let assert = require('assert')
 
 // Write a function called double that doubles a number
 
+const double = (num) =>{
+  return num*2
+}
+
 // Uncomment out the next line to test your solution
-// runQ1Tests()
+runQ1Tests()
 
 // Question Two:
 
 // Write a function called containsSeven that returns whether or not a number has an sevens in it.
 
+const containsSeven = (num) =>{
+  return num.toString().includes("7")
+}
 // Uncomment out the next line to test your solution
-// runQ2Tests()
+runQ2Tests()
 
 // Question Three:
 
 // Write a function called capitalizeTheAs the capitalizes all of the lowercase "a"s in a string leaving all other characters the same
 
+const capitalizeTheAs = (string) =>{
+  let newString= ""
+  for( let i=0; i < string.length; i++){
+    if(string[i] === "a") {
+      newString += string[i].toUpperCase()
+    } else{
+      newString += string[i]
+    }
+  }
+  return newString
+}
+
 // Uncomment out the next line to test your solution
-// runQ3Tests()
+runQ3Tests()
 
 // Question Four:
 // Write a function called largest that returns the largest value in an array
 
+const largest = (arr) =>{
+  let largestNum = -Infinity
+  for(let i=0; i < arr.length; i++){
+    if(largestNum < arr[i]){
+      largestNum = arr[i]
+    }
+  }
+  return largestNum
+}
+
 // Uncomment out the next line to test your solution
-// runQ4Tests()
+runQ4Tests()
 
 // Question Five:
 // Write a function called average that returns the average of an array
 
+const average = (arr) =>{
+  let sum = 0
+  let average = 0
+  arr.forEach(e =>{
+    sum += e
+  })
+  return average = sum/arr.length
```

Comment:
since you're returning on the same line as you're updating the value of average, you can just `return sum/ arr.length`, then you wouldn't need the average variable.